github/PaginatedList.py

```diff
@@ -270,5 +276,29 @@ def get_page(self, page: int) -> List[T]:
         if self.__list_item in data:
             self.__totalCount = data.get("total_count")
             data = data[self.__list_item]
+        return [
+            self.__contentClass(self.__requester, headers, self._transformAttribute(element), completed=False)
+            for element in data
+        ]
 
-        return [self.__contentClass(self.__requester, headers, element, completed=False) for element in data]
+    @classmethod
+    def attributes_transformer_override_from_dictionary(
+        cls, transformer: Callable[[Dict[str, Any]], Dict[str, Any]], overrides: dict
+    ) -> Callable[[Dict[str, Any]], Dict[str, Any]]:
```

Comment:
Why do we need this extra `transformer`? Can we simplify this to

```suggestion
    def override_attributes(cls, overrides: Dict[str, Any]) -> Callable[[Dict[str, Any]], Dict[str, Any]]:
```

and then use it as

```python
attributesTransformer=PaginatedList.override_attributes({"secrets_url": f"{self.url}/actions/secrets"}),
```